src/script/cpp_api/s_base.cpp

```diff
@@ -159,19 +159,25 @@ int ScriptApiBase::luaPanic(lua_State *L)
 
 void ScriptApiBase::clientOpenLibs(lua_State *L)
 {
-	static const std::vector<std::pair<std::string, lua_CFunction>> m_libs = {
-		{ "", luaopen_base },
-		{ LUA_TABLIBNAME,  luaopen_table   },
-		{ LUA_OSLIBNAME,   luaopen_os      },
-		{ LUA_STRLIBNAME,  luaopen_string  },
-		{ LUA_MATHLIBNAME, luaopen_math    },
-		{ LUA_DBLIBNAME,   luaopen_debug   },
 #if USE_LUAJIT
-		{ LUA_JITLIBNAME,  luaopen_jit     },
+#define M_LIBS_SIZE 7
```

Comment:
This optimization is somewhat useless. It's called once per session.

Reply:
@SmallJoker, fix it https://github.com/minetest/minetest/pull/12704/commits/7b18f1da0cf9bc70b334d51fc03ca93000e70811